src/Neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -465,6 +465,9 @@ public StackItem ToStackItem(ReferenceCounter referenceCounter)
                 NetworkFee,
                 ValidUntilBlock,
                 Script,
+
+                // Transaction signers
+                new Array(referenceCounter, Signers.Select(u => ((IInteroperable)u).ToStackItem(referenceCounter)))
```

Comment:
This will affect every `getTransaction` and `GetScriptContainer` call. It seems to be compatible (just another array item), but at the same time I fear we'll break something. Other than that it's OK to me.

Reminds me of #2657 and #2685.